lighthouse-core/audits/byte-efficiency/offscreen-images.js

```diff
@@ -117,11 +132,13 @@ class OffscreenImages extends ByteEfficiencyAudit {
       return results;
     }, new Map());
 
-    // TODO(phulce): move this to always use lantern
```

Comment:
why not move this to lantern like the other ones?

Reply:
oh this is moving it to be lantern-based 👍 

the extra check and comment below is clarifying that the timestamp dance is just an *extra* check in the non-lantern case, the graph should cover us for the most part

Reply:
got it. Comment + code (lantern artifact doesn't have `timestamp`) was a little confusing, I guess.

Maybe add something about `timestamp` like ``// If we're in the Lantern case and `timestamp`  isn't available, we  just have to rely...`

Reply:
good call 👍